src/geo/map/torque-layer.js

```diff
@@ -44,10 +49,18 @@ var TorqueLayer = LayerModelBase.extend({
   },
 
   _torqueTimeAttributeCartoCSSPropChanged: function () {
-    var regexp = /-torque-time-attribute:\s*["'](\w*)["']/;
-    var previousTorqueTimeAttributeProp = this.previous('cartocss').match(regexp) && this.previous('cartocss').match(regexp)[1];
-    var newTorqueTimeAttributeProp = this.get('cartocss').match(regexp) && this.get('cartocss').match(regexp)[1];
-    return newTorqueTimeAttributeProp !== previousTorqueTimeAttributeProp;
+    var currentCartoCSS = this.get('cartocss');
+    var previousCartoCSS = this.previous('cartocss');
+    var currentValue = this._getPropertyValueFromCartoCSS(currentCartoCSS, this.TORQUE_TIME_ATTRIBUTE_PROP);
```

Comment:
> if you check more than one attrs you will be parsing

I don't understand what you mean by that...

I'm adding the rest of the attributes and I think I I'll need to parse both the new and old cartocss (so parsing will only be done twice)...